### DISCUSSION TITLE
Dependabot noise over "version bump only" updates

### DISCUSSION THREAD

**Discussion by jeubank12** [aws/aws-sdk-js-v3#4562] [Q&A]
Are there any recommended mitigations in a node project to quiet down dependabot. The (imho too) frequent release schedule includes no-op changes to most of the modules. In less than a week, you can have 5 version numbers change over 5 packages and all of them have no relevant changes

If no mitigation, has there been thought to changing the versioning to not be global or perhaps a slower channel to subscribe to?

**Comment by mbrevda** [comment]
see also: https://github.com/aws/aws-sdk-js-v3/issues/6781

**Comment by jeubank12** [comment]
This is somewhat mitigated by dependabot groups configuration https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference#groups--

**Reply by mbrevda** [reply]
thanks - I'm using renovatebot
